bin/docs.test.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -ev
+
+# Make sure we use the correct repo root dir
+DIR=`dirname $0`
+REPO_ROOT=$DIR/..
+pushd $REPO_ROOT >/dev/null
+rm -rf sandbox/loopback.io/
+# Shadow clone is faster
+git clone --depth 1 https://github.com/strongloop/loopback.io.git sandbox/loopback.io
+npm run bootstrap
```

Comment:
IIUC, all other packages have been already bootstrapped and all we need is to bootstrap `sandbox/loopback.io`. To speed things up, can we tell lerna to bootstrap this single package only? Something along the following lines:

```
lerna bootstrap --scope loopback.io-workflow-scripts
```

Note that `loopback.io-workflow-scripts` is the package name as defined in loopback.io's package.json file:

https://github.com/strongloop/loopback.io/blob/cb67ad4c437503d7726845c09851b3256eceed16/package.json#L2

Reply:
I see, I did not realise `before_script` is being skipped. However, do we still need to bootstrap all packages to test the website? Isn't it enough to bootstrap `loopback.io` and perhaps `docs`?

Reply:
Good point! I wonder if bootstrapping only the loopback.io package is enough to make it bootstrap any packages that are dependencies (aka docs).